requirements/base.txt

```diff
@@ -69,15 +58,15 @@ geojson==2.5.0
 # pnnl/buildingid-py
 -e git+https://github.com/SEED-platform/buildingid.git@f68219df82191563cc2aca818e0b0fa1b32dd52d#egg=pnnl-buildingid
 
-oauthlib==2.0.2
-# cffi & cryptography dependencies needs to be installed via bitbucket/github due to a pip segfault with Python 3.6.7
-# -e hg+https://bitbucket.org/cffi/cffi@v1.11.5#egg=cffi
-# -e git+https://github.com/pyca/cryptography@2.5#egg=cryptography
+oauthlib==2.0.3
 
 # Used by jwt-oauth2
 django-braces==1.14.0
 
-jwt-oauth2>=0.1.1
-django-oauth-toolkit==1.0.0
+# Use forked jwt-oauth2 b/c the published version broke when updating to Django 3.x
+# See: https://docs.djangoproject.com/en/dev/releases/3.0/#removed-private-python-2-compatibility-apis
+# jwt-oauth2>=0.1.1
+-e git+https://github.com/macintoshpie/jwt-oauth2.git@f2db7e16803969f6121129a7b2fa04d09ec5f5e9#egg=jwt-oauth2
```

Comment:
🍴

Reply:
@nllong can you fork my edits to this repo into the SEED org so yall can maintain it?
https://github.com/macintoshpie/jwt-oauth2